components/nexus-mime/src/main/resources/org/apache/tika/mime/custom-mimetypes.xml

```diff
@@ -15,14 +15,4 @@
 -->
 <mime-info>
 
```

Comment:
ok, makes sense - is it worth adding a comment to the file explaining how to use it?